docker-compose.yml

```diff
@@ -17,7 +17,7 @@ services:
       - postgres
 #   output stat files to /dont/do/ in python, sync here into ./docker/pylo
     volumes:
-      - ./docker/pylo:/dont/do/
+      - ./output:/output
```

Comment:
It looks like this way doesn't work on linux, the directory is empty after app exits, i will search for other solutions, maybe just a different configuration of docker between mac and linux?

Reply:
maybe some permission issues?